sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -300,6 +303,35 @@ export interface RestErrorOptions {
     statusCode?: number;
 }
 
+// @public
+export class RetryError extends Error {
+    constructor(message?: string);
+    errors: RestError[];
+}
+
+// @public
+export function retryPolicy(...strategies: RetryStrategy[]): PipelinePolicy;
+
+// @public
+export interface RetryStrategy {
+    logger?: AzureLogger;
+    meetsConditions?(state: RetryStrategyState): boolean;
```

Comment:
To support the existing retry policies we should be able to skip them in case they match other retry policies.